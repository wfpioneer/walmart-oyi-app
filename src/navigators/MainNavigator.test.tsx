import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {
  MainNavigator
} from './MainNavigator';

describe('MainNavigator', () => {
  it('Renders the main navigator component', () => {
    const renderer = ShallowRenderer.createRenderer();
    renderer.render(
      <MainNavigator />
    );
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
