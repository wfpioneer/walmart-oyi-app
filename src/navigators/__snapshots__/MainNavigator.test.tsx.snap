// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNavigator Renders the main navigator component 1`] = `
<ForwardRef(NavigationContainerInner)>
  <StackNavigator
    screenOptions={
      Object {
        "headerShown": false,
      }
    }
  >
    <Screen
      component={[Function]}
      name="Login"
    />
    <Screen
      component={[Function]}
      name="Tabs"
    />
    <Screen
      component={[Function]}
      name="ReviewItemDetails"
    />
    <Screen
      component={[Function]}
      name="PrintPriceSign"
    />
    <Screen
      component={[Function]}
      name="PalletManagement"
    />
    <Screen
      component={[Function]}
      name="SettingsTool"
    />
    <Screen
      component={[Function]}
      name="Binning"
    />
    <Screen
      component={[Function]}
      name="Picking"
    />
    <Screen
      component={[Function]}
      name="WorklistNavigator"
    />
  </StackNavigator>
</ForwardRef(NavigationContainerInner)>
`;
